app/helpers/administrate/application_helper.rb

```diff
@@ -34,7 +34,7 @@ def svg_tag(asset, svg_id, options = {})
     end
 
     def sanitized_order_params
-      params.permit(:search, :id, :order, :page, :per_page, :direction)
+      params.permit(:search, :id, :order, :page, :per_page, :direction, :orders)
```

Comment:
Sorry, a bit late to the party but: was this change intended? Looks to me like it comes from "orders" being one of the examples, and it slipped through.

Reply:
🤔 I think you are right. I'll runt the tests without that and report back.

Reply:
OK, don't spend too long on this though! Sorry, I was looking at something else and this confused me and thought of commenting here.

Reply:
It fails:
```
1) customer show page displays the customers orders paginated when the second page is specified displays the second page of results
     Failure/Error: params.permit(:search, :id, :order, :page, :per_page, :direction)
     
     ActionView::Template::Error:
       found unpermitted parameter: :orders
```

Reply:
Thank you for looking into that 👍 

After your finding, I dug a bit deeper: it's a bug! `orders` is only relevant in that very specific example, where a "customer" has many "orders". The `orders` param tells the page number for the `orders` has_many list.

If I try to paginate through any other relationship, it breaks because the param has a different name.

No matter, I'll fix this.